scripts/generate/run.js

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+
+const {
+  toInt,
+  toUint,
+  toIntDownCast,
+  toUintDownCast,
+} = require('./templates');
+
+function genSafeCast (path) {
```

Comment:
For formatting, this script will come in handy: https://github.com/OpenZeppelin/contracts-wizard/blob/master/packages/core/src/utils/format-lines.ts